lib/sorbet-rails/model_plugins/active_record_factory_methods.rb

```diff
@@ -8,9 +8,9 @@ def generate(root)
 
     factory_methods = ['create', 'create!', 'new']
 
-    factory_methods.each do |factory_method|
-      add_factory_method(model_rbi, factory_method)
-    end
+    # factory_methods.each do |factory_method|
```

Comment:
I think we can just delete this file entirely?

Reply:
Yeah looks like this file is not needed because we can use `T.attached_class` for factory method